server/pkg/id/generator.go

```diff
@@ -56,12 +52,12 @@ func (g randGenerator) ID() ID {
 
 func (g randGenerator) TraceID() trace.TraceID {
 	tid := trace.TraceID{}
-	g.rand.Read(tid[:])
+	rand.New(rand.NewSource(time.Now().UnixNano())).Read(tid[:])
```

Comment:
A small suggestion (feel free to ignore!): we can isolate the seed from the generator for readability:
```suggestion
	rndSeed := rand.NewSource(time.Now().UnixNano())
	rand.New(rndSeed).Read(tid[:])
```